terraform/modules/pre-auth-lambda/lambda.tf

```diff
@@ -0,0 +1,14 @@
+resource "aws_lambda_function" "lambda_pre_auth" {
+  filename         = "${path.module}/${var.name_prefix}-pre-auth.zip"
```

Comment:
this should reference `archive_file.lambda_pre_auth_zip.output_path`

Reply:
This is done.